Donation-platform/src/main/java/com/google/DbCalls.java

```diff
@@ -238,11 +239,13 @@ public void updateCharity(Charity charity) throws Exception {
   }
   // Function takes modified class and updates database.
   public void updateTag(Tag tag) throws Exception {
-    Entity tagEntity = datastore.get(tag.getId());
-    tagEntity.setProperty(NAME, tag.getName());
-    tagEntity.setProperty(TRENDING_SCORE, tag.getTrendingScoreTag());
-    tagEntity.setProperty(IMGSRC, tag.getImgSrc());
-    datastore.put(tagEntity);
+    Query query = new Query(TAG).setFilter(new FilterPredicate("id", FilterOperator.EQUAL, tag.getId().getId()));
+    for (Entity tagEntity : datastore.prepare(query).asIterable()) {
```

Comment:
This for loop is implemented similar to getAllCharities in DbCalls. It'll probably run very slowly. Just FYI.

Reply:
yeah, I think we'll have to discuss how to optimize these queries. this change on my branch was just to get the update function to have the correct functionality required for updating tag scores.